src/services/Api.ts

```diff
@@ -403,14 +403,23 @@ export const getPod = (namespace: string, name: string) => {
   return newRequest<Pod>(HTTP_VERBS.GET, urls.pod(namespace, name), {}, {});
 };
 
-export const getPodLogs = (namespace: string, name: string, container?: string, sinceTime?: number) => {
+export const getPodLogs = (
+  namespace: string,
+  name: string,
+  container?: string,
+  tailLines?: number,
+  sinceTime?: number
+) => {
   const params: any = {};
   if (container) {
     params.container = container;
   }
   if (sinceTime) {
     params.sinceTime = sinceTime;
   }
+  if (tailLines && tailLines > 0) {
+    params.tailLines = tailLines;
+  }
```

Comment:
My only comment here is to exclude `params.sinceTime` if tailLines is not "all". I'm getting this filters combined by time and also number of lines. So, if you select "10 lines" but had previously selected "Last 1m", I sometimes get no logs.

Reply:
You're right, it isn't working well.  But after thinking about this a bit more I think I prefer both filters to be active.  I'm not going to disable the duration dropdown.  This way the user if free to play with either setting and by default will limit to 500 lines for the current duration set for the tabs.